POC-OOBE/OOBE-Sandbox/PowerToys Settings Sandbox/Views/UpdateContentDialog.xaml

```diff
@@ -0,0 +1,34 @@
+﻿<ContentDialog
+    x:Class="PowerToys_Settings_Sandbox.Views.UpdateContentDialog"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:PowerToys_Settings_Sandbox.Views"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    mc:Ignorable="d"
+    Title="What's New in PowerToys"
+    PrimaryButtonText="Got it"
+    Height="600"
+    >
+
+    <ScrollViewer>
+        <Grid>
+        <StackPanel>
+            <StackPanel Orientation="Vertical" HorizontalAlignment="Center">
+            <StackPanel Margin="0,12,0,0">
```

Comment:
Should I assume that this XAML is just a placeholder for dynamic content you will add later?

Reply:
Yes - placeholder for now, next steps: replace with dynamic content like with GridView